docs/build/qubit-order.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Bit-ordering in Qiskit
+description: Learn about Qiskit's ordering conventions and why we chose them
+---
+
+# Order of bits and qubits in Qiskit
+
+If you have a set of $n$ bits (or qubits), you'll usually label each bit $0
+\rightarrow n-1$. Different softwares and resources must choose how they order
+these bits both in computer memory and when displayed on screen.
+
+Here's how Qiskit orders bits in different scenarios:
+
+- The `QuantumCircuit` class stores its qubits in a list
+  (`QuantumCircuit.qubits`). The index of a qubit in this list defines the
+  qubit's label.
+
+  ```python
+  from qiskit import QuantumCircuit
+  qc = QuantumCircuit(2)
+  qc.qubits[0]  # qubit "0"
+  ```
+
+  ```
+  Qubit(QuantumRegister(2, 'q'), 0)
+  ```
+
+- On a circuit diagram, qubit $0$ is the topmost qubit, and qubit $n$ the
+  bottommost qubit (you can change this default with the `reverse_bits=True`
+  argument to `QuantumCircuit.draw`; see [Changing
+  ordering](#changing-ordering)).
+
+  ```python
+  qc.x(1)
+  qc.draw()
+  ```
+
+  ```
+  q_0: ─────
+       ┌───┐
+  q_1: ┤ X ├
+       └───┘
+  ```
+
+- When interpreting bits as a number, bit $0$ is the least significant bit, and
+  bit $n$ the most significant. This is helpful when coding because each bit
+  has the value $2^\text{label}$. For example, the following circuit execution
```

Comment:
As a total noob I don't understand the 2^label part... does "label" represent the bit's position?

Reply:
Ah, on my third reading I actually digested the part where you define "label" 😆 so I think nothing needs to change!

Reply:
Even if it's technically correct, do you think it could benefit from rephrasing?

Reply:
If you wanted to help people like me who don't retain that definition half-a-page later, you could add a parenthetical:
 

> 2^label (`label` being the qubit's index in the list).

Reply:
Will do, thanks! (And thanks for the review)